src/args.rs

```diff
@@ -198,6 +198,15 @@ pub struct CliArgs {
     #[arg(short = 'o', long = "overwrite-files", env = "OVERWRITE_FILES")]
     pub overwrite_files: bool,
 
+    /// Enable renaming files during file upload if duplicate exists
```

Comment:
This should give the user an indication of exactly _how_ they will be renamed.

Reply:
Done